entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -11,29 +11,106 @@ uid: core/what-is-new/ef-core-3.0/breaking-changes
 > [!IMPORTANT]
 > Please note that the feature sets and schedules of future releases are always subject to change, and although we will try to keep this page up to date, it may not reflect our latest plans at all times.
 
-## SQLite
+These are breaks in either API or behavior between the 2.2.x releases and the 3.0.0 release. Breaks in new features introduced from one 3.0 preview to another 3.0 preview are not documented here.
 
-* Microsoft.EntityFrameworkCore.Sqlite now depends on SQLitePCLRaw.bundle_e_sqlite3 instead of SQLitePCLRaw.bundle_green. This makes the version of SQLite used on iOS consistent with other platforms.
-* Removed SqliteDbContextOptionsBuilder.SuppressForeignKeyEnforcement(). EF Core no longer sends `PRAGMA foreign_keys = 1` when a connection is opened. Foreign keys are enabled by default in SQLitePCLRaw.bundle_e_sqlite3. If you're not using that, you can enable foreign keys by specifying `Foreign Keys=True` in your connection string.
+## High-impact breaks
+
+TODO: Removal of automatic client-evaluation of queries
```

Comment:
@ajcvickers @smitpatel what do you think we should have here? I think we should have something saying that the change isn't fully implemented yet but then explaining what it is.

Reply:
I don't think we need anything in the docs until we ship it

Reply:
Here's a crazy idea: switch off automatic client evaluation even before query 3.0 is ready...

Reply:
@ajcvickers I think that may be a good crazy idea. Let’s discuss it today.

Reply:
FWIW I think it's a great idea.